src/com/example/ui.cljs

```diff
@@ -92,33 +100,58 @@
+                     "Track how you feel so you know when to train.")
+                 (button :.self-center.text-grey-100.font-bold.bg-blue-600.py-3.px-8.rounded-md.border.border-transparent.hover:bg-indigo-700.h-12
+                         {:onClick #(github-signin)}
+                         "Sign in with GitHub")))
+       (div :.md:grid-span-1.md:m-auto.mx-4
+            (div :.flex.flex-col.text-grey-100.max-w-lg.h-full.place-content-center
+                 (h2 :.text-2xl.font-medium.mb-8.px-2
+                     "Your body is giving you the signals you need...")
+                 (p :.max-w-prose.mb-5.px-4
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")
+                 (p :.max-w-prose.mb-5.px-4
+                    "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.")
+                 (p :.max-w-prose.mb-5.px-4
+                    "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
+                 (h2 :.text-2xl.font-medium.mb-8.px-2
+                     "No fancy gear required, just answer these questions")
+                 (p :.max-w-prose.mb-5.px-4
+                    "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.")
+                 (p :.max-w-prose.mb-5.px-4
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")))))
+
+(dr/defrouter TopRouter [this props]
+  {:router-targets [Login Quiz]})
+
+(def ui-top-router (comp/factory TopRouter))
 
-              (div :.md:grid-span-1.md:m-auto.mx-4
-                   (div :.flex.flex-col.text-grey-100.max-w-lg.h-full.place-content-center
-                        (h2 :.text-2xl.font-medium.mb-8.px-2 "Your body is giving you the signals you need...")
-                        (p :.max-w-prose.mb-5.px-4 "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")
-                        (p :.max-w-prose.mb-5.px-4 "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.")
-                        (p :.max-w-prose.mb-5.px-4 "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
+(defsc Main [this {:main/keys [header router user] :as props}]
+  {:query [{:main/router (comp/get-query TopRouter)}
```

Comment:
the router is a statemachine that keeps track of which route its on. We have to query for all that data and pass it down into the router component.